javascript/ql/src/semmle/javascript/frameworks/UriLibraries.qll

```diff
@@ -94,10 +94,10 @@ module urijs {
 module uridashjs {
 
   /**
-   * Gets a data flow source node for the uridashjs library.
+   * Gets a data flow source node for member `name` of the uridashjs library.
    */
-  DataFlow::SourceNode uridashjs() {
-    result = DataFlow::moduleImport("uri-js")
+  DataFlow::SourceNode uridashjsMember(string name) {
```

Comment:
Can we make this predicate (and its brethren below) private?

Reply:
I can, but that will break all of the library-specific sanity tests.

Reply:
Hm, OK. Never mind, then.